test/test_mps.py

```diff
@@ -368,6 +368,12 @@ def _linear_helper(self, in_features, out_features, shape, bias=True, backward_p
                 self.assertEqual(cpu_linear.bias.grad.size(), mps_linear.bias.grad.size())
                 self.assertEqual(cpu_linear.bias.grad, mps_linear.bias.grad.to("cpu"), atol=8e-04, rtol=10.4e-05)
 
+    def test_linear1D(self):
+        self._linear_helper(in_features=2, out_features=3, shape=([2]), bias=True, backward_pass=False)
```

Comment:
This test case would have been automatically applied for you if MPS was using OpInfo testing:

```
def sample_inputs_linear(self, device, dtype, requires_grad, **kwargs):
    features_options = [[3, 4], [8, 8]]
    batch_options: List[List[int]] = [
        [],  # no batch
        [0],
        [8],
        [2, 3],
    ]
```

Please consider making the MPS tests use OpInfos!

Reply:
cc @mruberry

Reply:
Scaffolding for this just landed: https://github.com/pytorch/pytorch/pull/79532
There is still some work to reduce flakyness in MPS ops though.